### PR TITLE
Java 8 JCE link fixed

### PR DESCRIPTION
**Pull request by douglarek**

### CODE REVIEW

**Comment by pivotal-issuemaster** [comment]
@douglarek Please sign the [Contributor License Agreement](https://cla.pivotal.io/sign/spring?repositoryId=spring-cloud/spring-cloud-config&pullRequestId=513)!

[Click here](https://cla.pivotal.io/sync/spring?repositoryId=spring-cloud/spring-cloud-config&pullRequestId=513) to manually synchronize the status of this Pull Request.

See the [FAQ](https://cla.pivotal.io/faq) for frequently asked questions.

**Comment by pivotal-issuemaster** [comment]
@douglarek Thank you for signing the [Contributor License Agreement](https://cla.pivotal.io/sign/spring?repositoryId=spring-cloud/spring-cloud-config&pullRequestId=513)!

File: README.adoc [hunk 1]
```diff
@@ -242,7 +242,7 @@ http://www.oracle.com/technetwork/java/javase/downloads/jce-6-download-429243.ht
 
 http://www.oracle.com/technetwork/java/javase/downloads/jce-7-download-432124.html[Java 7 JCE]
 
-http://www.oracle.com/technetwork/java/javase/downloads/jce-7-download-432124.html[Java 8 JCE]
+http://www.oracle.com/technetwork/java/javase/downloads/jce8-download-2133166.html[Java 8 JCE]
```
Comment (spencergibb) [inline README.adoc]: According to the first line:

> // Do not edit this file (e.g. go instead to src/main/asciidoc)

**Comment by douglarek** [comment]
#515

**Pull request closed without merging**